doc/src/manual/methods.md

```diff
@@ -211,26 +211,21 @@ of methods:
 
 ```julia-repl
 julia> methods(+)
-# 180 methods for generic function "+":
-[1] +(x::Bool, z::Complex{Bool}) in Base at complex.jl:227
-[2] +(x::Bool, y::Bool) in Base at bool.jl:89
-[3] +(x::Bool) in Base at bool.jl:86
-[4] +(x::Bool, y::T) where T<:AbstractFloat in Base at bool.jl:96
-[5] +(x::Bool, z::Complex) in Base at complex.jl:234
-[6] +(a::Float16, b::Float16) in Base at float.jl:373
-[7] +(x::Float32, y::Float32) in Base at float.jl:375
-[8] +(x::Float64, y::Float64) in Base at float.jl:376
-[9] +(z::Complex{Bool}, x::Bool) in Base at complex.jl:228
-[10] +(z::Complex{Bool}, x::Real) in Base at complex.jl:242
-[11] +(x::Char, y::Integer) in Base at char.jl:40
-[12] +(c::BigInt, x::BigFloat) in Base.MPFR at mpfr.jl:307
-[13] +(a::BigInt, b::BigInt, c::BigInt, d::BigInt, e::BigInt) in Base.GMP at gmp.jl:392
-[14] +(a::BigInt, b::BigInt, c::BigInt, d::BigInt) in Base.GMP at gmp.jl:391
-[15] +(a::BigInt, b::BigInt, c::BigInt) in Base.GMP at gmp.jl:390
-[16] +(x::BigInt, y::BigInt) in Base.GMP at gmp.jl:361
-[17] +(x::BigInt, c::Union{UInt16, UInt32, UInt64, UInt8}) in Base.GMP at gmp.jl:398
+# 207 methods for generic function "+" from Base:
+   [1] +(x::T, y::T) where T<:Union{Int128, Int16, Int32, Int64, Int8, UInt128, UInt16, UInt32, UInt64, UInt8} @ int.jl:87
+   [2] +(c::Union{UInt16, UInt32, UInt64, UInt8}, x::BigInt) @ Base.GMP gmp.jl:529
+   [3] +(c::Union{Int16, Int32, Int64, Int8}, x::BigInt) @ Base.GMP gmp.jl:535
+   [4] +(c::Union{UInt16, UInt32, UInt64, UInt8}, x::BigFloat) @ Base.MPFR mpfr.jl:376
+   [5] +(c::Union{Int16, Int32, Int64, Int8}, x::BigFloat) @ Base.MPFR mpfr.jl:384
+   [6] +(c::Union{Float16, Float32, Float64}, x::BigFloat) @ Base.MPFR mpfr.jl:392
+   [7] +(x::Union{Dates.CompoundPeriod, Dates.Period}) @ Dates /Users/me/.julia/dev/julia/usr/share/julia/stdlib/v1.7/Dates/src/periods.jl:372
```

Comment:
Not sure what path should be shown here; this one is obviously from me building locally. (It's not a doctest.)

Reply:
On 1.7, these also don't look like the old docstring -- instead they have a path into the downloaded version, like `/Applications/Julia-1.7.app/.../LinearAlgebra/src/`:
```
julia> methods(+)
# 208 methods for generic function "+":
[1] +(x::T, y::T) where T<:Union{Int128, Int16, Int32, Int64, Int8, UInt128, UInt16, UInt32, UInt64, UInt8} in Base at int.jl:87
[2] +(c::Union{UInt16, UInt32, UInt64, UInt8}, x::BigInt) in Base.GMP at gmp.jl:529
[3] +(c::Union{Int16, Int32, Int64, Int8}, x::BigInt) in Base.GMP at gmp.jl:535
[4] +(c::Union{UInt16, UInt32, UInt64, UInt8}, x::BigFloat) in Base.MPFR at mpfr.jl:390
[5] +(c::Union{Int16, Int32, Int64, Int8}, x::BigFloat) in Base.MPFR at mpfr.jl:398
[6] +(c::Union{Float16, Float32, Float64}, x::BigFloat) in Base.MPFR at mpfr.jl:406
[7] +(x::Union{Dates.CompoundPeriod, Dates.Period}) in Dates at /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/stdlib/v1.7/Dates/src/periods.jl:372
[8] +(A::UpperHessenberg, B::UpperHessenberg) in LinearAlgebra at /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/stdlib/v1.7/LinearAlgebra/src/hessenberg.jl:99
```